scripts/README.md

```diff
@@ -2,10 +2,11 @@
 
 The scripts in these folder are meant to help the user/contributor to get up and running with the application fast.
 
-`bootstrap` - will point you towards the setup you will need before picking up this application
+[`bootstrap`](scripts/bootstrap)- will point you towards the setup you will need before picking up this application
```

Comment:
The path for `scripts` doesn't need to be here. Just [`bootstrap`](bootstrap) will suffice.
Same for the 3 links below